sigs.yaml

```diff
@@ -0,0 +1,17 @@
+sigs:
+  - name: "Semantic Conventions: K8s"
+    owner:
+      - name: "semconv-k8s-approvers"
+        github: semconv-k8s-approvers
+    project: "https://github.com/open-telemetry/community/blob/main/projects/k8s-semconv.md"
+    board: "https://github.com/orgs/open-telemetry/projects/114/views/1"
+    areas:
+      - area:k8s
+      - area:container
+    labels:
+      - accepting_contributions
+      - needs_prototyping
```

Comment:
do you think we'd have SIG-specific labels?  trying to think of a scenario and only some super-advanced things come up.  So wonder if we can postpone adding those until we need it?

Reply:
Yeah that's true. I wanted to somehow encode that when someone sends a PR adding conventions, but hasn't linked any implementation/prototyping we would flag that automatically. But I guess you are right, that's probably always our expectations so may not need this label.

The idea I had (which now makes less sense) is when conventions are marked stable, the sig may not need prototyping but that's wrong xD